src/promptflow/promptflow/azure/operations/_flow_operations.py

```diff
@@ -410,11 +408,7 @@ def _try_resolve_code_for_flow(cls, flow: Flow, ops: OperationOrchestrator, igno
                 return
             if flow._code_uploaded:
                 return
-
-            # TODO(2567532): backend does not fully support generate flow.tools.json from blob storage yet
```

Comment:
@zhengfeiwang please help check if we still need this logic

Reply:
this can be deprecated as it's a workaround during our switch to code-first.